src/main/java/com/werfen/report/service/PdfGridReportService.java

```diff
@@ -0,0 +1,50 @@
+package com.werfen.report.service;
+
+import net.sf.jasperreports.engine.*;
+import net.sf.jasperreports.engine.design.JasperDesign;
+import net.sf.jasperreports.engine.export.JRPdfExporter;
+import net.sf.jasperreports.export.SimpleExporterInput;
+import net.sf.jasperreports.export.SimpleOutputStreamExporterOutput;
+import net.sf.jasperreports.export.SimplePdfExporterConfiguration;
+import net.sf.jasperreports.export.SimplePdfReportConfiguration;
+
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class PdfGridReportService {
+
+    Logger log = Logger.getLogger(GridReportService.class.getName());
+
+    public void exportToPdf(String filePath, JasperDesign jasperDesign, Map<String, Object> parameters, JRDataSource ds) throws JRException {
```

Comment:
Just to avoid consumers of this class knowing about Jasper, this class should not throw exceptions and handle them inside.